lib/config/graphite.go

```diff
@@ -0,0 +1,323 @@
+/*
+Copyright 2024 Huawei Cloud Computing Technologies Co., Ltd.
```

Comment:
```suggestion
Copyright 2025 Huawei Cloud Computing Technologies Co., Ltd.
```


It should be opengemini and not huawei cloud?
cc @xiangyu5632